server/src/internalClusterTest/java/org/opensearch/discovery/StableClusterManagerDisruptionIT.java

```diff
@@ -87,43 +87,43 @@ protected Collection<Class<? extends Plugin>> nodePlugins() {
-        logger.info("---> legit elected master node={}", masterNode);
+        // Figure out what is the elected cluster-manager node
+        final String clusterManagerNode = internalCluster().getMasterName();
+        logger.info("---> legit elected cluster-manager node={}", clusterManagerNode);
 
-        // Pick a node that isn't the elected master.
-        Set<String> nonMasters = new HashSet<>(nodes);
-        nonMasters.remove(masterNode);
-        final String unluckyNode = randomFrom(nonMasters.toArray(Strings.EMPTY_ARRAY));
+        // Pick a node that isn't the elected cluster-manager.
+        Set<String> nonClusterManagers = new HashSet<>(nodes);
+        nonClusterManagers.remove(clusterManagerNode);
+        final String unluckyNode = randomFrom(nonClusterManagers.toArray(Strings.EMPTY_ARRAY));
 
-        // Simulate a network issue between the unlucky node and elected master node in both directions.
+        // Simulate a network issue between the unlucky node and elected cluster-manager node in both directions.
 
         NetworkDisruption networkDisconnect = new NetworkDisruption(
-            new NetworkDisruption.TwoPartitions(masterNode, unluckyNode),
+            new NetworkDisruption.TwoPartitions(clusterManagerNode, unluckyNode),
             NetworkDisruption.DISCONNECT
         );
         setDisruptionScheme(networkDisconnect);
         networkDisconnect.startDisrupting();
 
-        // Wait until elected master has removed that the unlucky node...
-        ensureStableCluster(2, masterNode);
+        // Wait until elected cluster-manager has removed that the unlucky node...
+        ensureStableCluster(2, clusterManagerNode);
 
-        // The unlucky node must report *no* master node, since it can't connect to master and in fact it should
+        // The unlucky node must report *no* cluster-manager node, since it can't connect to cluster-manager and in fact it should
         // continuously ping until network failures have been resolved. However
-        // It may a take a bit before the node detects it has been cut off from the elected master
+        // It may a take a bit before the node detects it has been cut off from the elected cluster-manager
         ensureNoMaster(unluckyNode);
 
         networkDisconnect.stopDisrupting();
 
-        // Wait until the master node sees all 3 nodes again.
+        // Wait until the cluster-manager node sees all 3 nodes again.
         ensureStableCluster(3);
 
-        // The elected master shouldn't have changed, since the unlucky node never could have elected itself as master
-        assertThat(internalCluster().getMasterName(), equalTo(masterNode));
+        // The elected cluster-manager shouldn't have changed, since the unlucky node never could have elected itself as cluster-manager
+        assertThat(internalCluster().getMasterName(), equalTo(clusterManagerNode));
     }
 
     private void ensureNoMaster(String node) throws Exception {
```

Comment:
`ensureNoMaster` -> `ensureNoClusterManager`?

Reply:
Thank you for taking your time in the careful review! 👍👍
I totally missed these method names in test classes, I will rename them in this PR.

Reply:
Some other method names:
For `assertMaster`, I changed its name in another PR: https://github.com/opensearch-project/OpenSearch/pull/2520/files#diff-8eef20b401000cc2fe94a8d5ee540ee6a77d15b0e6c9bdc2b6eb4da35e6f5dd3
For `assertNoMaster`, thank you for catching it, I forgot to change its name, and I just renamed it and updated the PR 2520.
For `blockMasterFromFinalizingSnapshotOnIndexFile`, `blockMasterOnWriteIndexFile`, and `blockMasterFromDeletingIndexNFile`, since they are public method in `test/framework` directory, I will rename them in the next phase, with other public Java API.

Reply:
Alright, I searched all the `master` reference in test method names by using regex `test(\w*)Master(\w*)\(\)` 🙂, nothing in the test method names should be missed any more. Haha

Reply:
About the folder name, such as https://github.com/opensearch-project/OpenSearch/tree/2.0.0-rc1/server/src/internalClusterTest/java/org/opensearch/action/support/master, I saw there is a corresponding folder in the source code https://github.com/opensearch-project/OpenSearch/tree/main/server/src/main/java/org/opensearch/action/support/master, so I was thinking to change the folder name along with renaming the public package names in the source code set.
For the method names in the class `InternalClusterTest`, I'm planning to rename the `master` terminology along with the other Java APIs, because those methods with `master` word are all public (https://opensearch.org/javadocs/2.0.0-rc1/OpenSearch/test/framework/build/docs/javadoc/org/opensearch/test/InternalTestCluster.html), I will rename them in the next major version.
The above 2 problems will be resolved in the issue https://github.com/opensearch-project/OpenSearch/issues/1684